Gemfile

```diff
@@ -47,5 +47,11 @@ group :development, :test do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+
+  gem 'faker'
```

Comment:
Funny gem - good research.
I'm usually making FactoryGirls sequences to make the diversification.